rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -294,31 +281,6 @@ def _to_binary_inst(msg):
         return msg
 
 
-def _to_time_inst(msg, rostype, inst=None):
-    # Create an instance if we haven't been provided with one
-
-    if rostype == "time" and msg == "now":
-        return ROSClock().now().to_msg()
-
-    if inst is None:
-        if rostype == "time":
-            inst = rclpy.time.Time().to_msg()
-        elif rostype == "duration":
-            inst = rclpy.duration.Duration().to_msg()
-        else:
-            return None
-
-    # Copy across the fields, try ROS1 and ROS2 fieldnames
-    for field in ["secs", "nsecs", "sec", "nanosec"]:
-        try:
-            if field in msg:
-                setattr(inst, field, msg[field])
-        except TypeError:
-            continue
```

Comment:
What's the impact of removing this? Does it mean a client publisher might need to change their code if they are still using the old values?

It's not necessarily bad to remove, but technically we should bump the package version to 2.0.0 if we remove features. Maybe it's easier to just keep it?

Reply:
> Does it mean a client publisher might need to change their code if they are still using the old values?

Yes, if this part is removed, they should change their code that uses `secs/nsecs`.

But I believe they should do so because they must update their subscriber-side code in any case, since `rosbridge` outputs Time data in the `sec/nanosec` format.
What do you think about it?

> It's not necessarily bad to remove, but technically we should bump the package version to 2.0.0 if we remove features.

I see, I agree with you!

> On the other hand, since some of the changes from #692 are significant to clients, I wonder if we should bump the version to 2.x anyway.

Yes, I think it's a good idea.